tests/tests_events_tree/test_events_tree.py

```diff
@@ -256,6 +256,15 @@ def test_get_root_by_id(general_data: List[dict]):
     assert collection.get_root_by_id("84db48fc-d1b4-11eb-b0fb-199708acc7bc") == tree
 
 
+def test_get_tree_by_id(general_data: List[dict]):
```

Comment:
Why do we use test_events_tree.py module to test ETC ?

Reply:
Fixed in https://github.com/th2-net/th2-data-services/pull/127/commits/883dc58dd7c048fd5e7397594791b551a330d36b
This test moved in test_events_tree_collection.py